Misc/NEWS.d/next/Core and Builtins/2022-12-09-14-27-36.gh-issue-100143.5g9rb4.rst

```diff
@@ -0,0 +1,3 @@
+When built with ``--enabled-pystats``, stats collection is now off by
```

Comment:
```suggestion
When built with ``--enable-pystats``, stats collection is now off by
```
(Haven't checked but that's the usual format)

Reply:
You're right.  This was just a typo here on my part.